libraries/standard/utilities/include/iot_logging_setup.h

```diff
@@ -0,0 +1,272 @@
+ *
+ * @return No return value.
+ */
+
+/**
+ * @def IotLogError( message  )
+ * @brief Abbreviated logging macro for stand-alone message strings at level #IOT_LOG_ERROR.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLogWithArgs( IOT_LOG_ERROR, "%s" , message )
+ * @endcode
+ */
+
+/**
+ * @def IotLogErrorWithArgs( pFormat, ...  )
+ * @brief Abbreviated logging macro for messages with arguments at level #IOT_LOG_ERROR.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_ERROR, pFormat, ... )
+ * @endcode
+ */
+
+/**
+ * @def IotLogWarn( message  )
+ * @brief Abbreviated logging macro for stand-alone message strings at level #IOT_LOG_WARN.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_WARN, "%s" , message )
+ * @endcode
+ */
+
+/**
+ * @def IotLogWarnWithArgs( pFormat, ...  )
+ * @brief Abbreviated logging macro for messages with arguments at level #IOT_LOG_WARN.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_WARN, pFormat, ... )
+ * @endcode
+ */
+
+/**
+ * @def IotLogInfo( message  )
+ * @brief Abbreviated logging macro for stand-alone message strings at level #IOT_LOG_INFO.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_INFO, "%s" , message )
+ * @endcode
+ */
+
+/**
+ * @def IotLogInfoWithArgs( pFormat, ...  )
+ * @brief Abbreviated logging macro for messages with arguments at level #IOT_LOG_INFO.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_INFO, pFormat, ... )
+ * @endcode
+ */
+
+/**
+ * @def IotLogDebug( message  )
+ * @brief Abbreviated logging macro for stand-alone message strings at level #IOT_LOG_DEBUG.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_DEBUG, "%s" , message )
+ * @endcode
+ */
+
+/**
+ * @def IotLogDebugWithArgs( pFormat, ...  )
+ * @brief Abbreviated logging macro for messages with arguments at level #IOT_LOG_DEBUG.
+ *
+ * Equivalent to:
+ * @code{c}
+ * IotLog( IOT_LOG_DEBUG, pFormat, ... )
+ * @endcode
+ */
+
+/* Check that LIBRARY_LOG_LEVEL is defined and has a valid value. */
+#if !defined( LIBRARY_LOG_LEVEL ) ||           \
+    ( ( LIBRARY_LOG_LEVEL != IOT_LOG_NONE ) && \
+    ( LIBRARY_LOG_LEVEL != IOT_LOG_ERROR ) &&  \
+    ( LIBRARY_LOG_LEVEL != IOT_LOG_WARN ) &&   \
+    ( LIBRARY_LOG_LEVEL != IOT_LOG_INFO ) &&   \
+    ( LIBRARY_LOG_LEVEL != IOT_LOG_DEBUG ) )
+    #error "Please define LIBRARY_LOG_LEVEL as either IOT_LOG_NONE, IOT_LOG_ERROR, IOT_LOG_WARN, IOT_LOG_INFO, or IOT_LOG_DEBUG."
+/* Check that LIBRARY_LOG_NAME is defined and has a valid value. */
+#elif !defined( LIBRARY_LOG_NAME )
+    #error "Please define LIBRARY_LOG_NAME."
+#else
+    #if LIBRARY_LOG_LEVEL != IOT_LOG_NONE
+        #if !defined( IotLog )
+            #error "Please define the common logging interface macro, IotLog(messageLevel, pFormat, ...)."
+        #endif
+    #endif
+
+    #if LIBRARY_LOG_LEVEL == IOT_LOG_DEBUG
+        /* All log levels will logged. */
+        #define IotLogError( pFormat )                 IotLog( IOT_LOG_ERROR, "%s", pFormat )
```

Comment:
```#define IotLogError( pFormat )``` --> ```#define IotLogError( message )``` 

Same for all the occurrences below.

Reply:
Fixed